test/files/ibp001/generate.py

```diff
@@ -0,0 +1,36 @@
+import os
+import ifcopenshell
+import ifcopenshell.template
+
+for validity, elem in [
+    ("pass", 'IfcRailing'),
+    ("pass", 'IfcSlab'),
+    ("pass", 'IfcStairFlight'),
+    ("pass", None),
+    ("fail", "IfcBeam"), 
```

Comment:
I would say a Beam should/could be allowed, same as an IfcMember.

Reply:
And what about IfcRoof and IfcWall, for instance? Should they be allowed? 

The documentation I found about it only mention flights, railings and slabs. e.g.; https://ifc43-docs.standards.buildingsmart.org/IFC/RELEASE/IFC4x3/HTML/lexical/IfcStair.htm